server/src/v1/Title/Service/title.service.ts

```diff
@@ -73,34 +72,30 @@ export class TitleService {
         return serializerService.serializeResponse('title_list', result)
     }
 
-    async createTitle(openedBy: string, payload: CreateTitleDto, buffer: Buffer): Promise<ISerializeResponse> {
+    async createTitle(openedBy: string, payload: any, buffer: Buffer): Promise<ISerializeResponse> {
         payload.name = payload.name.replace(/^\s+|\s+$/g, '')
         if (payload.name.length === 0) throw new BadRequestException('Title name can not be whitespace')
+        if (!payload.tags) throw new BadRequestException('tags must be provided')
```

Comment:
Following the [RFC definition of Unprocessable Entity](https://www.rfc-editor.org/rfc/rfc4918#section-11.2):
> The 422 (Unprocessable Entity) status code means the server
   understands the content type of the request entity (hence a
   415(Unsupported Media Type) status code is inappropriate), and the
   syntax of the request entity is correct (thus a 400 (Bad Request)
   status code is inappropriate) but was unable to process the contained
   instructions.

At this point, we understand the request, so it's not malformed.

Here the condition that tags must be present is more of validation.

Raise `UnprocessableEntityException` instead of the `BadRequestException` to provide as accurate error codes as possible.